content/zh-cn/docs/reference/kubernetes-api/authentication-resources/self-subject-review-v1.md

```diff
@@ -87,8 +87,11 @@ SelfSubjectReviewStatus 由 kube-apiserver 进行填充并发送回用户。
   - **userInfo.groups** ([]string)
 
     <!--
+    *Atomic: will be replaced during a merge*
+    
     The names of groups this user is a part of.
     -->
+    **原子性：合并期间将被替换**
```

Comment:
```suggestion

    **原子性：合并期间将被替换**
```
Need to add a blank line before the translations. See preview:
https://deploy-preview-48244--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/authentication-resources/self-subject-review-v1/